harness/determined/cli/job.py

```diff
@@ -57,14 +57,21 @@ def ls(args: Namespace) -> None:
             "Status",
             "User",
         ]
+
+        def computed_job_name(job: bindings.v1Job) -> str:
+            if job.type == bindings.determinedjobv1Type.TYPE_EXPERIMENT:
+                return f"{job.name} ({job.entityId})"
```

Comment:
why do we not need an entityID for non experiments?

Reply:
oh ops. i had automerge on.